demo/src/screens/componentScreens/ImageScreen.tsx

```diff
@@ -63,6 +64,7 @@ class ImageScreen extends Component {
               {renderBooleanOption.call(this, 'Show Overlay Content', 'showOverlayContent')}
               {renderRadioGroup.call(this, 'Overlay Type', 'overlayType', {none: 'none', ...Image.overlayTypes})}
               {renderSliderOption.call(this, 'Margin(margin-XX)', 'margin', {step: 4, min: 0, max: 40})}
+              {renderBooleanOption.call(this, 'Show Error Image', 'showErrorImage')}
```

Comment:
Move the Switch up together with the other toggles

Reply:
Done